pkg/yamlmeta/node.go

```diff
@@ -6,6 +6,7 @@ package yamlmeta
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/k14s/ytt/pkg/orderedmap"
```

Comment:
dont think we should be taking on orderedmap dependency from yamlmeta pkg.

Reply:
i guess we already do take it on: https://github.com/vmware-tanzu/carvel-ytt/blob/develop/pkg/yamlmeta/convert.go#L10

Reply:
I too went back-and-forth on this a bit.

- acknowledged: `yamlmeta` does not depend on the _behavior_ of `orderedmap.Map`
- the spirit of this PR is to shift from deny-listing values from Nodes to allow-listing. I believe that necessitates naming types explicitly, including `orderedmap.Map`.
- `orderedmap` is a utility-level package providing a generic data structure. As a dependency, while not massless, it's light.